src/Core/src/CommandMapper.cs

```diff
@@ -54,14 +54,27 @@ internal void Invoke(IElementHandler viewHandler, IElement? virtualView, string
 		public CommandMapper? Chained
 		{
 			get => _chained;
-			set
-			{
-				_chained = value;
-			}
+			set => _chained = value;
 		}
 	}
 
-	public class CommandMapper<TVirtualView, TViewHandler> : CommandMapper
+	public interface ICommandMapper
+	{
+		Command? GetCommand(string key);
+
+		void Invoke(IElementHandler viewHandler, IElement? virtualView, string property, object? args);
+	}
+
+	public interface ICommandMapper<out TVirtualView, out TViewHandler> : ICommandMapper
+		where TVirtualView : IElement
+		where TViewHandler : IElementHandler
+	{
+		void Add(string key, Action<TViewHandler, TVirtualView> action);
+
+		void Add(string key, Action<TViewHandler, TVirtualView, object?> action);
+	}
```

Comment:
This is the new interfaces, and the `out` modifiers are what allow some cool magic.